# Question

This code snippet makes a fetch to get the first todo item from the JSON placeholder API. Or at least it tries to.

What is wrong with this code? 

Describe how async/await helps us avoid this mistake.

```js
fetch('https://jsonplaceholder.typicode.com/todos/1')
  .then(response => {
    if (!response.ok) throw new Error('Fetch failed!');
    response.json();
  })
  .then(json => console.log(json))
  .catch(err => console.log(err))
```

You can run this code of course, but try not to! Try to figure out the issue on your own.

# Answer
    This code is wrong because we are not receiving the response.json() since we are not returning it; We can fix this using async/await by having variables to help determine whether things have been called. As well as we can catch an error separately from receiving our json().


# Teacher Feedback

Yup!

Score: 3/3